SQL/0000-00-00-schema.sql

```diff
@@ -1555,7 +1555,7 @@ CREATE TABLE participant_status (
         UserID varchar(255) default NULL,
         Examiner varchar(255) default NULL,
         entry_staff varchar(255) default NULL,
-        data_entry_date timestamp NOT NULL,
+        data_entry_date timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
@jacobpenny There should be an ALTER TABLE patch for existing projects too.

(You can ask someone else in the office today for details on how to do it/where to put it.)